lib/msteams_hermes/message.rb

```diff
@@ -62,15 +80,12 @@ def deliver
 
         response = http.request(req)
 
-        # For details see:
-        # https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/connectors-using?tabs=cURL%2Ctext1#send-messages-using-curl-and-powershell
-        if response.body != "1"
-          raise MessageBodyTooLargeError, body_json.bytesize if response.body.include? MSTEAMS_MESSAGE_413_ERROR_TOKEN
+        return response if MsTeamsWebhookType.mst_workflow_webhook_response?(response)
+        return response if MsTeamsWebhookType.mst_connector_webhook_response?(response)
 
-          raise UnknownError, response.body
-        end
+        raise MessageBodyTooLargeError, body_json.bytesize if response.body.include? MSTEAMS_MESSAGE_413_ERROR_TOKEN
```

Comment:
I perceive this as a bit inconsistent with the above early returns
Since above we are checking the content of `response.body` inside dedicated methods,
Here instead we are checking it directly in this method

Reply:
I see. That's true. I'll see what I can come up with.

Reply:
Addressed